.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,43 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: 'bug, needs triage'
+---
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is)
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/opctl/opctl/blob/master/CONTRIBUTING.md#reporting-new-issues)?
```

Comment:
maybe kill this section about Contributing Guidelines on issues; we don't have it and not convinced we need it.